File: widget/src/main/java/com/uploadcare/android/widget/exceptions/UploadcareWidgetException.java
package com.uploadcare.android.widget.exceptions;

public class UploadcareWidgetException extends RuntimeException {
    public UploadcareWidgetException(String message) {
        super(message);
    }

    public UploadcareWidgetException(String message, Throwable cause) {
        super(message, cause);
    }

    public UploadcareWidgetException(Throwable cause) {
        super(cause);
    }
}

File: widget/src/main/java/com/uploadcare/android/widget/exceptions/UploadcareWidgetException.kt
package com.uploadcare.android.widget.exceptions

/**
 * Error returned by uploadcare Widget.
 */
@Suppress("unused")
class UploadcareWidgetException : RuntimeException {
    constructor(message: String? = null) : super(message)
    constructor(cause: Throwable) : super(cause)
    constructor(message: String, cause: Throwable) : super(message, cause)
}